connect-examples/oauth/ruby/Gemfile.lock

```diff
@@ -1,102 +1,122 @@
+      faraday (>= 1, < 3)
+    faraday-gzip (0.1.0)
+      faraday (>= 1.0)
+      zlib (~> 2.1)
+    faraday-http-cache (2.4.1)
       faraday (>= 0.8)
-    faraday_middleware (0.14.0)
-      faraday (>= 0.7.4, < 1.0)
-    i18n (1.8.10)
+    faraday-multipart (1.0.4)
+      multipart-post (~> 2)
+    faraday-net_http (2.1.0)
+    faraday-net_http_persistent (2.0.2)
+      faraday-net_http (< 3)
+      net-http-persistent (~> 4.0)
+    faraday-retry (1.0.3)
+    i18n (1.12.0)
       concurrent-ruby (~> 1.0)
     little-plugger (1.1.4)
-    logging (2.3.0)
+    logging (2.3.1)
       little-plugger (~> 1.1)
       multi_json (~> 1.14)
-    loofah (2.9.1)
+    loofah (2.19.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
-    mini_portile2 (2.4.0)
-    minitest (5.14.4)
+    minitest (5.16.3)
     multi_json (1.15.0)
-    multipart-post (2.1.1)
-    mustermann (1.1.1)
+    multipart-post (2.2.3)
+    mustermann (2.0.2)
       ruby2_keywords (~> 0.0.1)
-    nokogiri (1.10.10)
-      mini_portile2 (~> 2.4.0)
-    rack (2.2.3)
-    rack-protection (2.0.8.1)
+    net-http-persistent (4.0.1)
+      connection_pool (~> 2.2)
+    nokogiri (1.13.8-x86_64-darwin)
+      racc (~> 1.4)
+    racc (1.6.0)
+    rack (2.2.4)
+    rack-protection (2.2.2)
       rack
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
+    rack-test (2.0.2)
+      rack (>= 1.3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.3.0)
+    rails-html-sanitizer (1.4.3)
       loofah (~> 2.3)
-    railties (5.2.6)
-      actionpack (= 5.2.6)
-      activesupport (= 5.2.6)
+    railties (6.1.7)
+      actionpack (= 6.1.7)
+      activesupport (= 6.1.7)
       method_source
-      rake (>= 0.8.7)
-      thor (>= 0.19.0, < 2.0)
-    rake (13.0.3)
-    ruby2_keywords (0.0.2)
-    sinatra (2.0.8.1)
-      mustermann (~> 1.0)
-      rack (~> 2.0)
-      rack-protection (= 2.0.8.1)
+      rake (>= 12.2)
+      thor (~> 1.0)
+    rake (13.0.6)
+    ruby2_keywords (0.0.5)
+    sinatra (2.2.2)
+      mustermann (~> 2.0)
+      rack (~> 2.2)
+      rack-protection (= 2.2.2)
       tilt (~> 2.0)
-    square.rb (6.0.0.20200625)
-      certifi (~> 2018.1)
-      faraday (~> 0.15)
-      faraday-http-cache (~> 2.0)
-      faraday_middleware (~> 0.13)
-      logging (~> 2.2)
-    thor (1.1.0)
-    thread_safe (0.3.6)
-    tilt (2.0.10)
-    tzinfo (1.2.9)
-      thread_safe (~> 0.1)
+    square.rb (22.0.0.20220817)
+      certifi (~> 2018.1, >= 2018.01.18)
+      faraday (~> 2.0, < 2.5.0)
+      faraday-follow_redirects (~> 0.2)
+      faraday-gzip (~> 0.1)
+      faraday-http-cache (~> 2.2)
+      faraday-multipart (~> 1.0)
+      faraday-net_http_persistent (~> 2.0)
+      faraday-retry (~> 1.0)
+      logging (~> 2.3)
+    thor (1.2.1)
+    tilt (2.0.11)
+    tzinfo (2.0.5)
+      concurrent-ruby (~> 1.0)
+    zeitwerk (2.6.1)
+    zlib (2.1.1)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-20
```

Comment:
Why no longer ruby?

Reply:
actually no idea, this was a generated file

Reply:
since v2.2* bundler no longer includes the `ruby` platform by default ([see conversation here](https://github.com/rubygems/rubygems/issues/4269)), so we'll have to add it ourselves.